src/nav2_tasks/include/nav2_tasks/ComputePathToPoseTaskMessages.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_TASKS__COMPUTEPATHTOPOSETASKMESSAGES_HPP_
+#define NAV2_TASKS__COMPUTEPATHTOPOSETASKMESSAGES_HPP_
+
+#include "nav2_tasks/msg/path_end_points.hpp"
+#include "nav2_tasks/msg/path.hpp"
+
+namespace nav2_tasks
+{
+
+using ComputePathToPoseCommand = nav2_tasks::msg::PathEndPoints;
```

Comment:
I suggest to rename ComputePathToPoseCommand to PathToPoseCommand since this is just a name for the command and the command itself is not performing the computation.

Reply:
Not sure if removing 'Compute' from the message name is better...

Reply:
I've used a consistent naming for the commands \<Action\>Command. Where action is a verb followed by any nouns. For example: Compute, Follow, Navigate, etc. So, you end up with \<Verb\>\<NounsThatVerbOperatesOn\>Command.